### ISSUE TITLE
 Tensorflow support rnn with batch normalzition cell? Http://arxiv.org/abs/1510.01378

### ISSUE THREAD

**Issue by guoying1030** [tensorflow/tensorflow#7514]
Tensorflow support rnn with batch normalzition cell? Http://arxiv.org/abs/1510.01378

**Comment by warchildmd** [comment]
These are the existing cells: https://www.tensorflow.org/api_docs/python/rnn_cell/rnn_cells_for_use_with_tensorflow_s_core_rnn_methods

You can create custom cells by extending `tf.nn.rnn_cell.RNNCell` and overwriting method `__call__(self, inputs, state, scope=None) `

**Comment by guoying1030** [comment]
@warchildmd 
@aselle 
When I use the __liner function, do a linear change, how to call batch normalztion, and use clip relu activation function?

**Comment by aselle** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
